.github/workflows/test.yml

```diff
@@ -9,22 +9,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        tarantool: ['1.10', '2.5', '2.6', '2.7', '2.8', '2.10']
+        tarantool: ['2.11', '3.3']
```

Comment:
Originally matrix resulted 7 jobs (coveralls=true for 2.11 and false for the rest ones). Now it results in - 2.11 + false
- 3.3 + false
- 2.11 + true
Do we really need this set? Do we understand the meaning of coveralls? Do we know the reason of the original exception for 2.11? I don't know the meaning of coveralls, but it doesn't look necessary to test both coveralls (true+false) for 2.11.